### PR TITLE
Article for distributing your PWA (4.3)

### PR DESCRIPTION
**Pull request by JudahGabriel**

### CODE REVIEW

File: docs/30DaysOfPWA/platforms-practices/03.md [hunk 1]
```diff
@@ -1,13 +1,119 @@
...
+
+Your PWA will be easier to install. While users can add your PWA to homescreen or install from the browser, many non-technical users are unfamilar with these tasks. Publishing your PWA to app stores lets users find and install your app using their normal workflow: app store ➡ search ➡ install. 
+
+You'll get better engagement from users as your app tile appears where users are looking: in their Start Menu and Taskbar (Windows), Dock (Mac), and home screen (Android and iOS).
+
+Users will be more likely to trust your app. An additional level of safety and security comes with apps that have gone through app store review processes.
+
+## Distribution model
+
+Hopefully I've piqued your interest in getting your PWA published to app stores.
```
Comment (zateutsch) [inline docs/30DaysOfPWA/platforms-practices/03.md:31]:
```suggestion
Hopefully, I've piqued your interest in getting your PWA published to app stores. 
```

File: docs/30DaysOfPWA/platforms-practices/03.md [hunk 2]
```diff
@@ -1,13 +1,119 @@
...
+In an ideal world, you'd submit your PWA's URL to the app store(s), and they'd publish your app. 
+
+But currently, app stores don't support PWAs natively. Instead, app stores require you to wrap your PWA in app packaging, making it look like a native app:
+
+- An `.msix` package (Windows)
+- An `.aab` package (Android)
+- An `.app` archive (iOS and Mac)
+- An `.apk` package (Oculus, Samsung, Amazon)
+
+Fortunately, there are free developer tools that create these app packages for you. Microsoft's free and open source [PWABuilder](https://www.pwabuilder.com) is such a tool. With PWABuilder, you input your PWA's URL, and PWABuilder will generate app packages for your desired app stores.
```
Comment (zateutsch) [inline docs/30DaysOfPWA/platforms-practices/03.md:50]:
```suggestion
Fortunately, there are free developer tools that create these app packages for you. Microsoft's free and open source [PWABuilder](https://www.pwabuilder.com) is such a tool. With PWABuilder, you input your PWA's URL, and PWABuilder will generate app packages for your desired app stores. We took a look at how to use PWA Builder in [this article](../dev-tools/07.md) from Week 3.
```

File: docs/30DaysOfPWA/platforms-practices/03.md [hunk 3]
```diff
@@ -1,13 +1,119 @@
...
+### 2. Register for a developer account
+
+Once you generate your app package, you'll need to create a developer account in order to submit your PWA for app store review:
+
+- For Windows, register with [Microsoft Partner Center](https://aka.ms/learn-pwa/30days-4.3/partner.microsoft.com)
+- For Android, register with [Google Play Console](https://aka.ms/learn-pwa/30days-4.3/play.google.com/console/about)
+- For iOS and Mac, register with [Apple Developer](https://aka.ms/learn-pwa/30days-4.3/developer.apple.com/account)
+- For Oculus, register with [Oculus Developer Center](https://aka.ms/learn-pwa/30days-4.3/developer.oculus.com/documentation/web/pwa-submit-app)
+- For Samsung, register with [Samsung Developer Center](hhttps://aka.ms/learn-pwa/30days-4.3/developer.samsung.com)
+- For Amazon, register with [Amazon Developer Portal](https://aka.ms/learn-pwa/30days-4.3/developer.amazon.com/apps-and-games)
```
Comment (zateutsch) [inline docs/30DaysOfPWA/platforms-practices/03.md:61]: Will be interesting to see metrics on how often each of these links are clicked.

File: docs/30DaysOfPWA/platforms-practices/03.md [hunk 4]
```diff
@@ -1,13 +1,119 @@
 # 4.3: Distributing Your PWA
 
-## *Coming soon!*
-
 ![Placeholder Banner Only. Replace when final assets ready.](_media/day3.png)
```
Comment (beth-panx) [inline docs/30DaysOfPWA/platforms-practices/03.md:3]:
```suggestion
![image of title and author.](_media/day3.png)
```

**Review by zateutsch — APPROVED** [review]

File: docs/30DaysOfPWA/platforms-practices/03.md [hunk 5]
```diff
@@ -1,13 +1,119 @@
...
+
+When I talk about publishing PWAs to app stores, devs often ask, "Why? Isn't the point of PWAs that you don't need an app store?"
+
+And they're right, you don't _need_ app stores. A great benefit of the web is there are no gatekeepers: you can publish your app without needing anyone's permission. Your app is truly your own, and you don't have to wait for app reviews to submit your app, add features to your app, or fix bugs. You don't have to share 30% of your app revenue with the app store.
+
+But app stores have their benefits too. Since the mobile computing explosion of the early 2000s, **we've trained a generation of users to look for apps in app stores.** And since Progressive Web Apps are, well, _apps_, why not list your app in the places where users go looking for apps? Your PWA becomes more discoverable -- and your app gains more users -- when you publish to app stores. 
+
+Your PWA will be easier to install. While users can add your PWA to homescreen or install from the browser, many non-technical users are unfamilar with these tasks. Publishing your PWA to app stores lets users find and install your app using their normal workflow: app store ➡ search ➡ install.
```
Comment (beth-panx) [inline docs/30DaysOfPWA/platforms-practices/03.md:23]:
```suggestion
Your PWA will be easier to install. While users can add your PWA to homescreen or install from the browser, many non-technical users are unfamiliar with these tasks. Publishing your PWA to app stores lets users find and install your app using their normal workflow: app store ➡ search ➡ install. 
```

File: docs/30DaysOfPWA/platforms-practices/03.md (hunk 2 above)
Comment (beth-panx) [inline docs/30DaysOfPWA/platforms-practices/03.md:50]:
```suggestion
Fortunately, there are free developer tools that create these app packages for you. Microsoft's free and open source [PWABuilder](https://aka.ms/learn-pwa/30days-4.3/pwabuilder.com) is such a tool. With PWABuilder, you input your PWA's URL, and PWABuilder will generate app packages for your desired app stores.
```

**Review by beth-panx — APPROVED** [review]
It would be nice if we can reference to [3.7](https://microsoft.github.io/win-student-devs/#/30DaysOfPWA/dev-tools/07) for publishing to Microsoft Store specific steps.

File: docs/30DaysOfPWA/platforms-practices/03.md [hunk 6]
```diff
@@ -1,13 +1,119 @@
...
+
+PWABuilder will analyze your PWA to ensure you have a manifest and a service worker:
+![image](https://user-images.githubusercontent.com/312936/156713592-74ed51c1-c736-4da7-9d66-762d1f671017.png)
+
+Advance to the publish page and choose `Windows` -> `Test Package`:
+![image](https://user-images.githubusercontent.com/312936/156716438-5780cdbc-d1d9-4ff0-8576-527f147a118f.png)
+
+(Why `Test Package`? This will let us test out our PWA on a Windows device before we publish to the Store.)
+
+This will download a zip file containing your app package and instructions for testing it on your Windows device. If all looks good, you're ready to [publish to the Microsoft Store](https://aka.ms/learn-pwa/30days-4.3/blog.pwabuilder.com/docs/publish-a-new-app-to-the-microsoft-store):
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/platforms-practices/03.md:101]: Worth prefacing this by saying something like "at this point you can publish to any of the app stores mentioned before, but let's look at what the process looks like for the Microsoft Store".

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**